test/todo_test.rb

```diff
@@ -1,13 +1,25 @@
 # frozen_string_literal: true
 
 require "test_helper"
+require "stringio"
 
 class TodoTest < Minitest::Test
   def test_that_it_has_a_version_number
     refute_nil ::Todo::VERSION
   end
 
-  def test_it_does_something_useful
-    assert false
+  def test_add_adds_item
+    test_file = StringIO.new()
```

Comment:
This is just a nit, but typically even those that stick to parens don't use bare parens for method invocations here.
```suggestion
    test_file = StringIO.new
```